src/main/java/com/artipie/rpm/CliArguments.java

```diff
@@ -64,6 +64,16 @@
         .hasArg()
         .build();
 
+    /**
+     * FileLists option.
+     */
+    private static final Option FILE_LISTS = Option.builder("f")
+        .argName("fl")
+        .longOpt("file-list")
```

Comment:
@marceloamadeu as "filelists" is the name of xml metadata file, I think it would be more clear if the option name would be exactly the same - `filelists` or with "need" prefix `need-filelists`. Let's better ask `arc`, @g4s8 wdyt?

Reply:
@marceloamadeu  I agree with @olenagerasimova let's call it just `filelists`

Reply:
@olenagerasimova and @g4s8 Done. Check please.